### PR TITLE
type safe sorts in Jakarta Data

### PR DESCRIPTION
**Pull request by njr-11**
Updates Open Liberty to align with the specification proposal for type-safe Sort.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/io.openliberty.jakarta.data.1.0/src/jakarta/data/page/Slice.java [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
+ * Copyright (c) 2022.2024 IBM Corporation and others.
```
Comment (mswatosh) [inline dev/io.openliberty.jakarta.data.1.0/src/jakarta/data/page/Slice.java]:
```suggestion
 * Copyright (c) 2022, 2024 IBM Corporation and others.
```

**Review by mswatosh — DISMISSED** [review]

**Comment by njr-11** [comment]
#build

**Review by mswatosh — APPROVED** [review]

**Comment by njr-11** [comment]
#libby

**Pull request merged**